### PR TITLE
dev/core#1190 Add pptx to safe file types

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
In the last security release we added a whitelist for uploadable file types - this missed some common types. We previously added ics & this add pptx

Before
----------------------------------------
pptx files cannot be uploaded

After
----------------------------------------
They are in the whitelist

Technical Details
----------------------------------------
I feel like it would be better to store these as an array somewhere - maybe as the first step to
a metadata construct for Option groups...

Comments
----------------------------------------
I guess some more of these will pop up - we can maybe treat any found within 6 months of the change as 'regressions'

https://lab.civicrm.org/dev/core/issues/1190

### CODE REVIEW

File: xml/templates/civicrm_data.tpl [hunk 1]
```diff
@@ -677,6 +677,7 @@ VALUES
   (@option_group_id_sfe, 'xlsx', 13, 'xlsx',  NULL, 0, 0, 13, NULL, 0, 0, 1, NULL, NULL, NULL),
   (@option_group_id_sfe, 'odt',  14, 'odt',   NULL, 0, 0, 14, NULL, 0, 0, 1, NULL, NULL, NULL),
   (@option_group_id_sfe, 'ics',  15, 'ics',   NULL, 0, 0, 15, NULL, 0, 0, 1, NULL, NULL, NULL),
+  (@option_group_id_sfe, 'pptx',  16, 'pptx',   NULL, 0, 0, 16, NULL, 0, 0, 1, NULL, NULL, NULL),
```
Comment (seamuslee001) [inline xml/templates/civicrm_data.tpl]: @eileenmcnaughton i think we would need to run ./bin/regen.sh as we are changing what should be in civicrm_generated.mysql file

Comment (eileenmcnaughton) [reply xml/templates/civicrm_data.tpl]: @seamuslee001 ok - updated

**Comment by seamuslee001** [comment]
@pfigel are you ok with this?

**Comment by eileenmcnaughton** [comment]
Note that per the ticket this should be the same risk level as docx - ie. 'std Microsoft risk level' ....

**Comment by pfigel** [comment]
> Note that per the ticket this should be the same risk level as docx - ie. 'std Microsoft risk level' ....

Agreed.

**Comment by seamuslee001** [comment]
Merging based on Patrick's comment and the fact we already list docx

**Pull request merged**